src/duration.rs

```diff
@@ -221,7 +221,9 @@ impl Duration {
         }
     }
 
-    /// Returns the number of nanoseconds such that
+    /// Returns the number of nanoseconds in the fractional part of the duration.
```

Comment:
These improvements for the new methods should be introduced in the same commit as the methods themselves, as I explained in my previous comment.